cmd/kubelet/app/options/globalflags.go

```diff
@@ -84,9 +84,7 @@ func addCredentialProviderFlags(fs *pflag.FlagSet) {
 	global := pflag.CommandLine
```

Comment:
nit: no need to get global here and pass to `addLegacyCloudProviderCredentialProviderFlags `. It can be taken when needed.